uncertainty_mapping/draw_uncertainty.py

```diff
@@ -184,4 +219,4 @@ def filter_points(points, min_distance=3):
         )
 
     # Save the map to an HTML file
-    m.save(args.output_filename)
+    m.save("output/" + args.output_filename + ".html")
```

Comment:
Thank you for the comment! 😃

The `runtime.sh` and `dev.sh` scripts ensure that the output directory is created (if necessary) before running Docker. They also mount it to the appropriate directory within the container that matches the script's scope.

Reply:
I can update the script to avoid hardcoding the `output` directory while preserving the previous behaviour. Happy to look into it after the holiday period—in the meantime, I’ll mark this PR as a draft :)